apps/src/lib/config/mod.rs

```diff
@@ -95,7 +95,7 @@ pub struct Ledger {
     pub genesis_time: Rfc3339String,
     pub chain_id: ChainId,
     pub shell: Shell,
-    pub tendermint: Tendermint,
+    pub cometbft: Tendermint,
```

Comment:
since this is already a breaking change, let's also rename the `Shell` `tendermint_dir` and `TENDERMINT_DIR` const